utils/build-warehouse.bash

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Update the Athena datawarehouse tables with the latest Causal generated DDL
+# You will need the aws command to be configured with appropriate permissions
+
+# This causes the script to fail and exit if any error occurs
+set -euo pipefail
+
+# IMPORTANT!
+# set CAUSAL_SQL to the location of the generated SQL file
+# In this example, assumed to be causal.sql and assumed to run
+# In the directory where the generated sql file is
+CAUSAL_SQL=causal.sql
+
+# IMPORTANT!
+# set OUTPUT_LOCATION to the location where you want athena responses to be placed
+# the bucket below is an example and needs to be changed
+OUTPUT_LOCATION=s3://causal-getting-started/ddlexecution/
+
+# IMPORTANT!
+# set DB_NAME to your database name
+DB_NAME=getting_started
+
+# create the database
+# you only need to to do this once
+aws athena start-query-execution --query-string "create database $DB_NAME;" --result-configuration "OutputLocation=$OUTPUT_LOCATION"
+
+# temp directory to hold sql file for each statements
+# athena command line does not work with multiple statements
+TMP_DIR=$(mktemp -d)
+
+# split file into separate statements
+# causal uses ---- to separate statements in the file
```

Comment:
Add comment: Athena only allows you to process one query at a time.

Reply:
mentioned on line 29. Will improve and duplicate here.